tests/plugins/test_pandas_extensions.py

```diff
@@ -30,6 +32,19 @@ def df():
     yield pd.DataFrame({"foo": ["bar"]})
 
 
+def test_pandas_parquet(df: pd.DataFrame, tmp_path: pathlib.Path) -> None:
+    file_path = tmp_path / "sample_df.parquet.gzip"
+    writer = PandasParquetWriter(path=file_path, engine="pyarrow")
+    writer.save_data(df)
+
+    reader = PandasParquetReader(path=file_path, engine="pyarrow")
+    read_df, metadata = reader.load_data(pd.DataFrame)
+
+    assert read_df.shape == df.shape, "DataFrames do not match"
+    assert read_df.columns == df.columns
```

Comment:
Pro-tip: you probably want to use https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html

Reply:
Thanks :), yeah I looked into this. When data is read from parquet, the string(python) in the original df became an object in the read_df...so I needed to pass a 'check_dtype=False' in other to pass the test. 

Is it a workaround or does it sounds right?

Reply:
ah -- that's fine. Likely something to look into how to get them to match easily. But for this test we just want to check that it looks right.